modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -81,42 +111,100 @@ public function testCreateListPages() {
     $this->assertNotNull($title_input);
     $this->assertNotNull($publication_date_input);
 
-    // Filter results by date.
+    // Filter the News results by date.
     $publication_date_input->setValue('gt');
     $date_input = $filter_form->findField('Date');
     $date_input->setValue('2022-04-04');
     $search_button->click();
-    $this->assertSearchResultsTitle(8);
+    $this->assertSearchResultsTitle('News', 6);
```

Comment:
As suggested below, let's pass the entire node title to the method.
```suggestion
    $this->assertSearchResultsTitle('News list page', 6);
```